src/newrelic/processor_test.go

```diff
@@ -181,6 +182,18 @@ var (
 	}
 )
 
+func GetLogEventAppInfo(limit int) AppInfo {
```

Comment:
The **Get** in the function name suggests that a value of a struct field will be returned but this function seems to create/make/build a new AppInfo structure with LogEvent limit. Therefore I would suggest a different name `BuildAppInfoWithLogEventLimit` to better indicate its purpose.

Reply:
That is a good suggestion - fixed in commit [3bd6d67](https://github.com/newrelic/newrelic-php-agent/pull/497/commits/3bd6d67c7ab67a11c26ed97bc95b19d685d32509).